steps/update-jira-release/0.0.1/step.yml

```diff
@@ -0,0 +1,89 @@
+title: Jira Release Status Update
+summary: Updates the release status in Jira, marking it for release. The step includes
+  the option to set the release as a draft.
+description: This step automates Jira release management by updating the release status.
+  It allows seamless marking of a release for deployment, with an optional draft setting.
+  Ensure your Jira instance details, release ID, authentication token, and other necessary
+  inputs are correctly configured for smooth integration with your CI/CD workflows.
+website: https://github.com/Ahmadalsofi/bitrise-step-update-jira-release
+source_code_url: https://github.com/Ahmadalsofi/bitrise-step-update-jira-release
+support_url: https://github.com/Ahmadalsofi/bitrise-step-update-jira-release/issues
+published_at: 2024-01-27T21:42:57.656659+03:00
+source:
+  git: https://github.com/Ahmadalsofi/bitrise-step-update-jira-release.git
+  commit: 9a4d87219533557f02246f5b68025d81f64bfcba
+host_os_tags:
```

Comment:
It is not causing any issues as is, but `host_os_tags` can be safely removed.

Reply:
Hey @lpusok , since I changed the name of the repository, I have opened a new pull request with the updated changes. 

new PR: https://github.com/bitrise-io/bitrise-steplib/pull/4094

Reply:
Meant to ask for a change in the actual title, not the repository (the repository is fine after the changes) here: https://github.com/bitrise-io/bitrise-steplib/pull/4083/files#r1474044575.

Reply:
@lpusok I changed the repository name for consistency